_about_livecoms/04_donations.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: single
+sidebar:
+  nav: donations.md
+title: Donating to LiveCoMS 
+excerpt: How to donate to LiveCoMS
+permalink: /about/donations
+---
+
+### Donations to LiveCoMS
+
+  We very much appreciate financial donations to LiveCoMS! Donations
+  are made to the CU Foundation of University of Colorado Boulder, the
+  official publisher of the journal, to a fund exclusively for use of
+  LiveCoMS and controlled by the managing editors. The CU Foundation
+  is a 501(c)(3) charitable organization, and donations are thus
+  usually tax-deductible.
+ 
+Follow the link felow to donate:
+
+  <p><a href="https://giving.cu.edu/fund/living-journal-computational-molecular-science-support-fund" class="btn ">Donate Now to LiveCoMS!</a></p>
```

Comment:
Is the `class="btn "` tag functional?

Reply:
supposed to be may need to test in live version.